odo/backends/sql.py

```diff
@@ -607,8 +607,7 @@ def append_select_statement_to_sql_Table(t, o, bind=None, **kwargs):
 
     query = t.insert().from_select(o.columns.keys(), o)
 
-    with bind.connect() as conn:
-        conn.execute(query)
+    bind.execute(query)
```

Comment:
Why does this no longer need to be in the begin context?

Reply:
each thing executed is in its own mini-transaction unless explicitly in a transaction already, this was redundant.